File: leetcode/java/reverse-vowels-of-a-string.java
import java.util.*;
class Solution{
    public String reverseVowels(String s){
        char[] word = s.toCharArray();
        String vowels = "aeiouAEIOU";
        int l = 0, r = s.length()-1;
        while (l<r) {
            while(l<r && vowels.indexOf(word[l])==-1){
                l++;
            }
            while(r>l && vowels.indexOf(word[r])==-1){
                r--;
            }
            char temp = word[l];
            word[l] = word[r];
            word
        }
    }
}

File: leetcode/python/reverse-vowels-of-a-string.py
"""
345. Reverse Vowels of a String
https://leetcode.com/problems/reverse-vowels-of-a-string/

Given a string s, reverse only all the vowels in the string and return it.

The vowels are 'a', 'e', 'i', 'o', and 'u', and they can appear in both cases.

Example 1:
    Input:  s = "IceCreAm"
    Output: "AceCreIm"
    Explanation: The vowels in s are ['I', 'e', 'e', 'A']. On reversing the
    vowels, s becomes "AceCreIm".

Example 2:
    Input:  s = "leetcode"
    Output: "leotcede"

Constraints:
    - 1 <= s.length <= 3 * 10^5
    - s consists of printable ASCII characters.
"""
class Solution:
    @staticmethod
    def isVowel(c):
        vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']
        if(c in vowels):
            return True
        return False
    def reverseVowels(self, s: str) -> str:
        arr = []
        for c in s:
            
